pkg/odo/cli/component/common_push.go

```diff
@@ -55,6 +57,16 @@ func (cpo *CommonPushOptions) InitConfigFromContext() error {
 	return nil
 }
 
+//InitEnvInfoFromContext initializes envinfo from the context
+func (cpo *CommonPushOptions) InitEnvInfoFromContext() error {
+	var err error
```

Comment:
Can remove this line by changing the return to `(err error)`

Reply:
updated